tests/performance-tests/src/tests/public-api/data-set-query/logging.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-console */
+import { RefinedResponse } from 'k6/http';
```

Comment:
Formatting looks a bit off in this file. Not sure if intentional?

Reply:
Will take a look, but it's all been through `npm run format`

Reply:
Yeah this is the work of the formatter I'm afraid :)